jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyManagerTest.java

```diff
@@ -84,25 +83,8 @@ public void assemblyFiles(
         .asList().isEmpty();
   }
 
-
-  @Test
-  public void testCopyValidVerifyGivenDockerfile(@Injectable final KitLogger logger) throws IOException {
-    BuildConfiguration buildConfig = createBuildConfig();
-
-    AssemblyManager.verifyAssemblyReferencedInDockerfile(
-        new File(getClass().getResource("/docker/Dockerfile_assembly_verify_copy_valid.test").getPath()),
-        buildConfig, new Properties(),
-        logger);
-
-    // @formatter:off
-    new Verifications() {{
-      logger.warn(anyString, (Object []) any); times = 0;
-    }};
-    //@formatter:on
-  }
```

Comment:
I'm not able to find this test. Has this test been deleted?

Reply:
these tests seem to have been parameterized into: AssemblyManagerVerifyAssemblyReferencedInDockerfileTest